### ISSUE TITLE
Update multiple domains

### ISSUE THREAD

**Issue by StevePearson-AIT** [oznu/docker-cloudflare-ddns#79]
I'm currently using the following docker-compose to start this container:
  cf-ddns:
    container_name: cf-ddns
    image: oznu/cloudflare-ddns:latest
    environment:
      - PUID=${PUID}
      - PGID=${PGID}
      - TZ=${TIME_ZONE}
      - API_KEY=${CLOUDFLARE_API_TOKEN}
      - ZONE=${DOMAINNAME}
      - PROXIED=true
    restart: always

This works fine but I now need to add a second domain at the same time. This is not a sub domain, it's completely different.
Is it possible to do this with the same container, or do I need to spin up a second instance to update the new domain?
Thanks.

**Comment by weirlive** [comment]
Also wondering this, I added 2 -ZONE and it will just update the 2nd one.

**Comment by mediacowboy** [comment]
Also intrested in this as it would be perfect for updating my 10 domains.

**Comment by kdpuvvadi** [comment]
Better sollution would be launch another container for another record or zone.

**Comment by StevePearson-AIT** [comment]
@kdpuvvadi I'm not sure about that, while I know it's not a huge footprint eating loads of resources, a single instance doing all of your domains would be a much better and neater solution. In my case I am currently only using 2 domains, but others have a lot more domains to update, for example @mediacowboy has 10 domains, so would need to run 10 instances, which would be very messy!

**Comment by kdpuvvadi** [comment]
> @kdpuvvadi I'm not sure about that, while I know it's not a huge footprint eating loads of resources, a single instance doing all of your domains would be a much better and neater solution. In my case I am currently only using 2 domains, but others have a lot more domains to update, for example @mediacowboy has 10 domains, so would need to run 10 instances, which would be very messy!

What I  normally do is set one domain With Arecord and make all others cname to the first one. That's actually recommend to me by data centre engineers.

**Comment by StevePearson-AIT** [comment]
@kdpuvvadi yeah, I looked at doing exactly this, and initially it seemed ok, but I hit problems. I can't remember exactly what the issue was, something to do with oauth I think, but when I pointed the second domain's A-Record at the name of the first domain some things stopped working correctly.

**Comment by mediacowboy** [comment]
I ended up going with dd client as it allows you to updated multiple a records for multiple domains.

**Comment by ikkerus** [comment]
The fun part is, that this was working on unraid with SUB-Domains at a specific point of release. Now it only updates the latest subdomain (subdomain6). Never testet with another full domain.

![image](https://user-images.githubusercontent.com/67939/152559905-c12e5852-3e3e-4d7b-b97d-5a2ac34fba74.png)
